inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/ClickHouseService.java

```diff
@@ -99,10 +99,13 @@ private void processOutput() {
             return;
         }
         // output
-        try (PreparedStatement pstat = this.conn.prepareStatement(INSERT_SQL)) {
```

Comment:
Suggested using the `try-with-resource`, because the `PreparedStatement` implements the `AutoCloseable` interface, it will be closed automatically, we don't need to write additional `finally` block to close it.

Reply:
Ok, you are right